docs/Tecnologías/Herramientas/vscode.md

```diff
@@ -0,0 +1,61 @@
+# ¿Que es VSCode?
+Visual Studio Code (VSCode) es un editor de código fuente gratuito y de código abierto desarrollado por Microsoft. Es compatible con diversos lenguajes de programación y ofrece una amplia gama de características para facilitar el desarrollo de aplicaciones.
+
+# Características principales
+- IntelliSense: VSCode ofrece autocompletado y sugerencias de código para varios lenguajes de programación.
+- Depuración: VSCode incluye herramientas de depuración para ayudar a encontrar y solucionar errores en el código.
+- Git integrado: VSCode incluye funcionalidades de control de versiones integradas para trabajar con repositorios Git.
+- Extensiones: VSCode cuenta con una gran cantidad de extensiones disponibles para personalizar y ampliar sus funcionalidades.
+
+# Descarga e instalación
+VSCode se puede descargar de forma gratuita desde la página web oficial de Microsoft: https://code.visualstudio.com/. La instalación es sencilla y seguir las instrucciones del asistente de instalación.
+
+# Configuración
+
+Al principio puede parecer extraño, pero todas las configuraciones de vscode estan en un archivo `.json` que pueden personalizar. Para ingresar a el tienen que teclear `[CTRL] + [SHIFT] + P` o `F1` y escribir `Preferencias: Abrir Configuración (JSON)` o `Preferences: Open Settings (JSON)` Si lo tienen en ingles.
```

Comment:
[Adding keyboard keys - Material for MkDocs](https://squidfunk.github.io/mkdocs-material/reference/formatting/?h=keyboard#adding-keyboard-keys)

Reply:
agregado en el commit [85e3cd2](https://github.com/open-source-uc/open-wiki/pull/1/commits/85e3cd2677cf3b585f538b66e3fbd26072b5d678)